engine/src/test/java/org/camunda/bpm/engine/test/api/history/BatchHistoricDecisionInstanceDeletionTest.java

```diff
@@ -626,6 +628,24 @@ public void shouldSetInvocationsPerBatchType() {
     configuration.setInvocationsPerBatchJobByBatchType(new HashMap<>());
   }
 
+  @Test
+  public void shouldSetExecutionStartTimeInBatchAndHistory() {
+    ClockUtil.setCurrentTime(TEST_DATE);
+
+    // when
```

Comment:
I'm missing the `// given` block.

Reply:
Fixed, restructured the code a little bit.